sapl/base/views.py

```diff
@@ -937,11 +937,16 @@ def get_context_data(self, **kwargs):
             return context
 
         qtdes = {}
-        for tipo in TipoMateriaLegislativa.objects.all():
-            qs = context['object_list']
-            qtde = len(qs.filter(tipo_id=tipo.id))
-            if qtde > 0:
-                qtdes[tipo] = qtde
+        materias_resultado = []
+        for obj in context['object_list']:
+            if obj not in materias_resultado:
+                materias_resultado.append(obj)
```

Comment:
```suggestion
        from collections import OrderedDict
        materias_resultado = list(OrderedDict.fromkeys(context['object_list']))
```

O código acima deve deduplicar as matérias ao mesmo tempo em que preserva a ordem da lista original.

Reply:
Implementado.